app/controllers/factories_controller.rb

```diff
@@ -0,0 +1,29 @@
+class FactoriesController < ApplicationController
+
+  def show
+
+  end
+
+  def new
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+    @factory = Factory.new
+  end
+
+  def create
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+
+    @factory = @manufacturer.factories.create(factory_params)
+    if @factory.valid?
+      redirect_to manufacturer_path(@manufacturer)
```

Comment:
redirect_to manufacturer_url(@manufacturer), notice: "New factory has been created in #{@factory.city}, #{@factory.state}"